sqlitecluster/SQLite.cpp

```diff
@@ -598,6 +598,14 @@ bool SQLite::prepare(uint64_t* transactionID, string* transactionhash) {
         _mutexLocked = true;
     }
 
+    // We pass the journal number selected to the handler so that a caller can utilize the
+    // same method bedrock does for accessing 1 table per thread, in order to attempt to
+    // reduce conflicts on tables that are written do on every command
+    _journalName = _journalNames[_sharedData.nextJournalCount++ % _journalNames.size()];
+    if (_shouldNotifyPluginsOnPrepare) {
+        (*_onPrepareHandler)(*this, SToInt64(SAfter(_journalName, "journal")));
```

Comment:
Hmmm actually, since the number of tables in the journal and in the onyxUpdates might not be the same, I am not sure we want/need to pass the tableID, I think it's better to send `_sharedData.nextJournalCount++`? That way we can do the same % computation with the onyx updates tables..